janus_core/src/message.rs

```diff
@@ -815,7 +898,7 @@ impl Decode for HpkeCiphertext {
 
         Ok(Self {
             config_id,
-            encapsulated_context,
+            encapsulated_key: encapsulated_context,
```

Comment:
nit: rename variable `encapsulated_context` -> `encapsulated_key`

Reply:
👍🏻 (also highlighted another spot where we need to use `{encode,decode}_u32_items` once we upgrade to a version of libprio that includes it)